extensions/jdk-logger-monitor/src/main/java/org/eclipse/dataspaceconnector/logger/LoggerMonitor.java

```diff
@@ -60,8 +61,10 @@ private void log(final Supplier<String> supplier, final Level level, final Throw
         if (errors == null || errors.length == 0) {
             LOGGER.log(level, supplier);
         } else {
-            Arrays.stream(errors).forEach(error -> LOGGER.log(level, supplier.get(), error));
+            String logMessage = Optional.ofNullable(supplier.get())
```

Comment:
Just out of curiosity: why did CodeQL complain here?

Reply:
It complained about Log injection
https://owasp.org/www-community/attacks/Log_Injection